gen3_util/common/__init__.py

```diff
@@ -0,0 +1,142 @@
+import csv
+import logging
+import pathlib
+from typing import Mapping, Iterator, Dict, TextIO
+from urllib.parse import urlparse
+
+import orjson
+import yaml
+from pydantic.json import pydantic_encoder
+import io
+import gzip
+from gen3_util.config import Config
+
+
+def print_formatted(config: Config, output: Mapping) -> None:
+    """Print the output, using configured output format"""
+
+    if config.output.format == "yaml":
+        print(yaml.dump(output))
+    elif config.output.format == "json":
+        print(
+            orjson.dumps(
+                output, default=pydantic_encoder, option=orjson.OPT_INDENT_2
+            ).decode()
+        )
+    else:
+        print(output)
+
+
+def read_ndjson_file(path: str) -> Iterator[dict]:
+    """Read ndjson file, load json line by line."""
+    with _file_opener(path) as jsonfile:
+        for l_ in jsonfile.readlines():
```

Comment:
Small syntax question -- what cases would the underscore following a variable name (e.g. `l_` as above) be used?